src/potential/2b/energy2b.cpp

```diff
@@ -22,19 +32,44 @@ double get_2b_energy(std::string m1, std::string m2, size_t nm,
 double get_2b_energy(std::string m1, std::string m2,size_t nm, 
                      std::vector<double> xyz1, std::vector<double> xyz2,
                      std::vector<double> &grd1, std::vector<double> &grd2) {
+  // Order the two monomer names and corresponding xyz
+  bool swaped = false;
   if (m2 < m1) {
     std::string tmp = m1;
     m1 = m2;
     m2 = tmp;
+    std::vector<double> tmp2 = xyz1;
```

Comment:
Same `std::move` comment. It should be doing the exact same thing, but always nice to be verbose about these things IMO.